Create index.html file and add portfolio html skeleton to index.html

simplegoose opened the pull request:
- Create an index.html file.
- Add HTML skeleton for my portfolio project.
- Link future CSS files in the head tag using the link tag.

Features:

1. Header and logo for mobile view.
2. Hero section.

simplegoose committed:
Create index.html file and add portfolio html skeleton to index.html

simplegoose committed:
On index.html
- Add new line at the end of the file

ha-manel reviewed (commented):
# STATUS: INVALID 🔴 
Hi 
I'm sure you put a lot of effort into this project but this PR is invalid because you have failed linters checks😓 

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

simplegoose committed:
Add css folder containing style.css file

simplegoose committed:
On css/style.css
- Add css rule to set box sizing to all element

Gopxfs commented:
# STATUS: INVALID REQUEST

Hi 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that doesn't contain a README file 😉. Please make sure to add one using [Microverse Template](https://github.com/microverseinc/readme-template), filled with your information.

Your Code Review Request will be marked invalid in your Dashboard, so please submit a new one once ready 🙏.
Invalid requests don't count as a review, so don't worry about that 😄!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Felix45 reviewed (changes requested):
Hi @simplegoose , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: 

You have submitted a project, that does not have a valid readme file. Please use this template from [Microverse](https://github.com/microverseinc/readme-template) [CLick Here >>](https://github.com/microverseinc/readme-template) to add a professional readme file to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

simplegoose committed:
Add readme file

roshan-bajgain reviewed (commented):
Hi @simplegoose, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a different pull request. This is `Portfolio: setup and mobile first` but you submitted different project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏a

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

simplegoose committed:
On index.html and On css/style.css
Feature: Header, hamburger menu and logo
- On index.html edit html skeleton to have a text My Logo on logo div.
- Comment out future code to be edited when adding new features
On css/style.css
- Import fonts and add it to font face
- Colors for project
- Style header, logo & hamburger

simplegoose committed:
On assets/
- Add font poppins to project
- Add vector images and hero background image

simplegoose committed:
On index.html
Feature: Hero section
-  Add hero section with styling done with css
- Add socials on hero section with assets imported from assets/vectors

simplegoose committed:
On css/style.css
- Lint style.css

joskalenda commented on index.html:
```diff
@@ -0,0 +1,217 @@
+</head>
+<body>
+  <div class="wrapper">
+    <header class="header">
+      <div class="logo">My Logo</div>
+      <div class="hamburger-menu">
```
- [x] I appreciate you have created this logo name section, but remember to change this text to something relevant to you as your name since this portfolio is owned by you, what do you think about it?
- [x] Remember to wrap the logo name in an `a` tag, this is to allow the user to navigate to the home page.
- [x] One more thing, for semantics reasons, this div on line 13 up to line 17 should be a `nav` tag instead, kindly change it.

simplegoose replied:
Hello @joskalenda 

Thank you for your review, I have wrapped my navigation inside a nav tag. The hamburger-menu is for creating the menu bars.

joskalenda commented on index.html:
```diff
@@ -0,0 +1,217 @@
+        </ul>
+        <span class="close">+</span>
+      </div>
+    </nav> -->
```
- [x] We Understand you wanted to keep this portion of code by commenting on it for later usage, but in a professional manner, this is not recommended, it does not make your code clean, please remove it and make sure to use git history if you need to see you previous commit codes.

joskalenda commented on index.html:
```diff
@@ -0,0 +1,217 @@
+            <button type="submit" class="buttons">Get In Touch</button>
+          </form>
+        </div>
+      </section> -->
```
- [x] Same as here, kindly remove it.

joskalenda commented on index.html:
```diff
@@ -0,0 +1,217 @@
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
```
<img width="330" alt="whaaaa" src="https://user-images.githubusercontent.com/77843050/205431188-a97cdbed-9775-4f5f-808e-1f3374293ac2.png">

- [ ] Do we see the same thing? 👀... this is what we are welcome with when we visit your page, please make sure you are well importing your CSS file in the HTML and resubmit your code with the correct design.

simplegoose replied:
Hello @joskalenda 

The design is working well on my local. I have no idea why it looks that way on your local. Can you please explain to me how you run the project?
Also, can you verify for me if the changes I made are okay? I have one review left. Thank you.

joskalenda reviewed (changes requested):
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @simplegoose , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
| <img src="https://media4.giphy.com/media/ReyD4OdRiHeLDCRaEP/giphy.gif"/> |

## Highlights

- [ ]...Great use of microverse readme template✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

simplegoose committed:
On index.html
- Uncoment code to be used in the future

simplegoose committed:
On css/style.css
- Set nav.flex-nav to display: none; in order to keep hero section in its place

simplegoose committed:
On index.html
- Wrap logo inside an anchor tag that links to hero section
On css/style.css
- Style anchor tag to have no decoration

simplegoose committed:
Change file path in index.html

binhussen commented on index.html:
```diff
@@ -0,0 +1,219 @@
+    <title>Timothy | Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
```
- [Optional] Kindly. a `<div>` wrapper above `<header>` is not necessary for semantic reasons. z👌😍

binhussen commented on index.html:
```diff
@@ -0,0 +1,219 @@
+        <div></div>
+      </div>
+    </header>
+    <nav class="flex-nav">
+      <div class="background-filter"></div>
+      <div class="nav-items-wrapper">
+        <ul>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+        <span class="close">+</span>
+      </div>
+    </nav>
```
- [optional] Good job, for preparing the navigation for later use 👍 👍  I think the `<nav>` is inside the `<header>` as it is part of the header.😘😘

binhussen commented on index.html:
```diff
@@ -0,0 +1,219 @@
+      <a href="#home">
+        <div class="logo">Timothy</div>
+      </a>
+      <div class="hamburger-menu">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
```
- [Required] Kindly, The hamburger button has a `nav` tag for semantics reasons. as your previous code reviewer suggested you 👌👌😍

binhussen reviewed (changes requested):
Hi @simplegoose , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media0.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- Kindly, make your hamburger in `nav` for a semantic reasons.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

simplegoose committed:
Add icon-menu to project
Wrap icon-menu in nav tag

jerryowusu reviewed (approved):
Hi @simplegoose , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![good job](https://user-images.githubusercontent.com/65233860/205465218-6a8caaed-8119-4606-a288-f40a70ef6cf0.gif)


## Highlights
All linters are passing :heavy_check_mark: 
Great effort in finishing this milestone :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
